docs/topics/spiders.rst

```diff
@@ -145,6 +145,41 @@ scrapy.Spider
        :param kwargs: keyword arguments passed to the :meth:`__init__` method
        :type kwargs: dict
 
+   .. classmethod:: update_settings(settings)
+
+       The ``update_settings()`` method is used to modify the spider's settings
+       and can be called during initialization of a spider instance.
+
+       It takes a :class:`~scrapy.settings.Settings` object as a parameter and
+       adds or updates the spider's configuration values. This method is a class method,
+       meaning that it is called on the :class:`~scrapy.Spider` class and allows all instances
+       of the spider to share the same configuration.
+
+       To create class hierarchies for spiders, it is recommended to use the :attr:`custom_settings`
+       attribute instead of ``update_settings()``, as it allows for default settings to be
+       defined and automatically inherited by subclasses.
```

Comment:
You're right that both methods are inherited similarly. To avoid confusion, I'll remove that paragraph and add a new one that highlights the benefits of using `update_settings()`